doc/userguide/rules/payload-keywords.rst

```diff
@@ -401,49 +401,53 @@ When ``relative`` is included, there must be a previous ``content`` or ``pcre``
 The result can be stored in a result variable and referenced by
 other rule options later in the rule.
 
```

Comment:
The sentence above and the array below should be kept together, right ?

Reply:
Yes -- will update.